core/src/main/java/io/jenkins/pluginhealth/scoring/model/updatecenter/SecurityWarning.java

```diff
@@ -26,9 +26,16 @@
 
 import java.util.List;
 
+import io.jenkins.pluginhealth.scoring.model.Resolution;
+
 public record SecurityWarning(
     String id,
+
     String name,
+
+    String url,
+
+    Resolution resolution,
```

Comment:
This is not what I meant.
You can use the `Resolution` in the `SecurityScoring` to explain why the scoring is bad. But the resolution is not parsed from the update-center file.

Reply:
Thanks @alecharp then i have to use Resolution to update the scoring and keep the probe changes same as i did in the previous commits

Reply:
Correct.

Reply:
Thanks :) Working on it